OrbitBase/Tracing.cpp

```diff
@@ -9,54 +9,69 @@
 
 #include "OrbitBase/Logging.h"
 #include "OrbitBase/Profiling.h"
+#include "OrbitBase/ThreadPool.h"
 
 using orbit::tracing::Scope;
 using orbit::tracing::ScopeType;
 using orbit::tracing::TimerCallback;
 
-thread_local std::vector<Scope> scopes;
+ABSL_CONST_INIT static absl::Mutex global_callback_mutex(absl::kConstInit);
+ABSL_CONST_INIT static TimerCallback* global_callback;
```

Comment:
So how is this set? Other PR?

Reply:
Yes, a variation of #1112. Essentially, we'll listen for introspection scopes and send them over grpc in the same way we send FunctionCalls in the collector.